dspace-api/src/main/java/org/dspace/content/Collection.java

```diff
@@ -336,4 +337,17 @@ private CollectionService getCollectionService() {
         return collectionService;
     }
 
+    /**
+     * return count of the collection items
+     *
+     * @return int
+     */
+    public int countArchivedItem() {
```

Comment:
Tiny thing, but this method should probably be plural `countArchivedItems` instead of `countArchivedItem`.  But, if you'd rather leave as-is, that's fine.

Reply:
Done.